### ISSUE TITLE
a+ | Captcha a11y user story #6038

### ISSUE THREAD

**Issue by fredewe** [liqd/adhocracy-plus#1924]
**URL:** https://aplus-dev.liqd.net/accounts/signup/?next=/
**user:** unregistered user with activated screen reader
**expected behaviour:** The screen reader reads all relevant information i need to register on adhocracy+
**behaviour:** Not all relevant things are read by the screen reader, there is information missing or the SR does not read it like we want it 
**important screensize:**-
**device & browser:** MacOS and Safari using MacOS VoiceOver and Chrome using Screen Reader extension 
**Comment/Question:** 
I don't know how to deal with this because i don't know if i can test it the way a person using SR would like it to be.
There is a screenshot where i included the thinks i have noticed.
Screenshot?
![Bildschirmfoto 2022-06-15 um 16 11 38](https://user-images.githubusercontent.com/5470479/173851605-10f3c4b3-5294-4c25-ba2d-96dfc7721316.png)

**Comment by phillimorland** [comment]
@fredewe The image version is not accessible for SR only text version, adding images names in the code means that bot just has to word match which is very simple also would mean we would have to use a different captcha which was not within the points and as far as opensource captcha's go, this is already one of the better ones.

The helptext is not tabbable as it is not the form label, it is only readable, this is a different function on SR's, normally `alt shift arrow`. It should however be focusable when there is a form error which is not a functionality we have anywhere and appears to be very difficult in django, there is a solution here which maybe @fuzzylogic2000 or @Rineee can check? my very quick attempt at implementation failed.

The not reading it - as math is definitely a problem and i'll check it,

Let me know if anything wasn't clear or if you have any questions.

**Comment by fredewe** [comment]
@phillimorland thank you very much for clarification, that helped me a lot understanding the possibilities we have here! If the reading as math is somehow possible it would be great.

**Comment by phillimorland** [comment]
further conversation about django forms and a11y here https://groups.google.com/g/django-updates/c/eRJbEvjka6U

**Comment by phillimorland** [comment]
@fredewe just noticed the other issue about the links not being read, that's not an issue I've come across with any of my screen readers, can you check are they also not tabbed to?

**Comment by phillimorland** [comment]
NOTE: link reading is only an issue in safari with voiceover, works in ff and chrome

**Comment by fredewe** [comment]
@phillimorland Thanks for investigating! That is fine for me!
